hw/ip/otp_ctrl/data/BUILD

```diff
@@ -341,6 +341,23 @@ otp_json(
     ],
 )
 
+otp_json(
```

Comment:
Should we get rid of the equivalent OTP configuration in //sw/device/silicon_creator/rom_ext/e2e/BUILD ?

Reply:
Yeah, I thought you did some refactoring in the splice PR. I will wait until that PR lands before updating the targets with `otp` dependencies.